### DISCUSSION TITLE
How do I override the MaxPageSize via appsettings OrchardCore_Navigation PagerOptions

### DISCUSSION THREAD

**Discussion by daveblack101** [OrchardCMS/OrchardCore#13639] [Q&A] [answered]
I tried adding the following section to appsettings but MaxPageSize is staying at 100.
```
"OrchardCore_Navigation": {
      "PagerOptions": {
        "PageSize": 50,
        "MaxPageSize": 300,
        "MaxPagedCount": 100
      }
    }
```

**Comment by jtkech** [comment, marked as answer]
These options are post configured by code from site settings.

https://github.com/OrchardCMS/OrchardCore/blob/74fbeea745d2c5443d11b992d0c5ea69eaa0c211/src/OrchardCore.Modules/OrchardCore.Settings/PagerOptionsConfiguration.cs#L15-L22

Site settings wins over configuration, configuration are used if the OC.Settings module is not enabled. You may need to do another post configuration, e.g. from a module that depends on OC.Settings so that it runs after, and that would bind again these options from the configuration.

**Reply by daveblack101** [reply]
Thank you so much. I wasn't quite sure where I was going wrong.
